markets/perps-market/contracts/interfaces/IPerpsAccountModule.sol

```diff
@@ -135,4 +133,18 @@ interface IPerpsAccountModule {
             uint256 requiredMaintenanceMargin,
             uint256 maxLiquidationReward
         );
+
+    /**
+     * @notice Allows anyone to pay an account's debt
+     * @param accountId Id of the account.
+     * @param amount debt amount to pay off
+     */
+    function payDebt(uint128 accountId, uint256 amount) external;
```

Comment:
Hey @sunnyvempati can you please tell me why this function was designed this way? Is there any blocker of having the debt absorbed by LP's and the collateral as well, instead of having the liquidator repay the debt and accordingly receive the collateral?